play_motion:
  motions:
    head_look_around:
      joints: [head_1_joint, head_2_joint]
      points:

      - positions: [0, 0]
        time_from_start: 0.1
      - positions: [-0.6, 0.2]
        time_from_start: 2.0
      - positions: [-0.6, -0.4]
        time_from_start: 4.0
      - positions: [0, 0.3]
        time_from_start: 6.0
      - positions: [0.6, 0.2]
        time_from_start: 8.0
      - positions: [0.6, -0.4]
        time_from_start: 10.0
      - positions: [0, 0]
        time_from_start: 12.0     

      meta:
        name: head_look_around
        usage: demo
        description: 'head_look_around'
